coalib/bears/PackageRequirement.py

```diff
@@ -19,3 +19,24 @@ def check(self):
         Check if the requirement is satisfied.
         """
         raise NotImplementedError
+
+
+class PythonRequirement(PackageRequirement):
+
+    def __init__(self, package, version=""):
+        PackageRequirement.__init__(self, 'pip', package, version)
+
+    @classmethod
+    def multiple(cls, *args):
+        reqs = ()
+        for requirement in args:
+            if isinstance(requirement, str):
+                reqs += (requirement)
```

Comment:
does it work if i do ``reqs += (requirement,)`` ? if not, how should I add the strings then?